src/tableaus/rkc_tableaus.jl

```diff
@@ -0,0 +1,1592 @@
+struct ROCK2ConstantCache{T,T2} <: OrdinaryDiffEqConstantCache
+  ms::SVector{46, Int}
+  fp1::SVector{46, T}
+  fp2::SVector{46, T}
+  recfi::SVector{4476, T2}
```

Comment:
This should be a Vector. You typically don't want to use `SVector` for large arrays. All of these should probably be arrays, but some testing would need to be done on the `46`.